pkg/policy/policy.go

```diff
@@ -33,7 +36,17 @@ type DefaultRuleDefinition struct {
 }
 
 func GetDefaultRules() (*DefaultRulesDefinitions, error) {
-	defaultRulesDefinitions, err := yamlToStruct(defaultRulesYamlContent)
+	configDefaultRulesYamlContent, err := getDefaultRulesFromFile()
+
+	var defaultRulesDefinitions *DefaultRulesDefinitions
+	if err == nil {
+		defaultRulesDefinitions, err = yamlToStruct(configDefaultRulesYamlContent)
+		fmt.Println("Using default rules from file")
```

Comment:
are those 2 logs here on purpose? will they be removed?

Reply:
Nice catch, Thanks!